lesson16/src/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+	<meta charset="UTF-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>TerraceTechフロントエンドエンジニア養成所</title>
+	<link rel="stylesheet" href="./css/reset.css" />
+	<link rel="stylesheet" href="./css/styles.css" />
+</head>
+
+<body>
+	<section class="news tabs">
+		<h1>もりけん塾<span>news</span></h1>
+		<ul id="js-tab" class="tab-list" role="tablist">
+		</ul>
+	</section>
+
+	<script type="module" src="./js/main.js"></script>
```

Comment:
IMO
[<script> タグに async / defer を付けた場合のタイミング](https://qiita.com/phanect/items/82c85ea4b8f9c373d684)
It is safer to put it in the head tag instead of just before the body tag.
In such cases, the defer attribute should be added.
```suggestion
	<script type="module" src="./js/main.js" defer></script>
```

Reply:
Thank you for the very easy-to-understand materials!
I learned a lot.

I read this article and corrected the position of script.

[a33705c](https://github.com/nyattsu72/hanson-frontend-cource/pull/21/commits/a33705cee2e271528344b37a84cd2f32ab21f6c0)